applications/serial_lte_modem/doc/PPP_AT_commands.rst

```diff
@@ -128,20 +166,15 @@ Response syntax
 Testing on Linux
 ================
 
-You can test SLM's PPP on Linux by using the ``pppd`` command, though SLM must be compiled without CMUX because there is no widely available utility that allows multiplexing a device file on Linux.
-
-.. note::
-
-   If you have a utility that allows multiplexing a device file on Linux, you can use SLM's PPP with the ``pppd`` command through CMUX.
-   To do this, you must first set up the CMUX link.
-   Then, make sure to replace the device file argument in the ``pppd`` command with that of SLM's PPP channel, which will have been created by the CMUX utility.
-   See :ref:`SLM_AT_CMUX` for more information on SLM's CMUX.
+You can test SLM's PPP on Linux by using the ``pppd`` command.
+This section is for a configuration without CMUX.
+If you are using CMUX, see :ref:`slm_as_linux_modem` for more information on setting it up.
 
```

Comment:
@SeppoTakalo Please check these new removals/changes related to CMUX in the documentation.

Reply:
My thinking was that we show here the basic use case. As this is more of an example (at least for me), we don't need to show all variations. Especially given that the change of a CID for PPP is a rare case.